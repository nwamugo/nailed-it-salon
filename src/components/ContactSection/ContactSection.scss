.contact-container .primary-header {
  color: #e294c5;
  text-align: center;
  text-transform: uppercase;
  font-family: "Josefin Sans", sans-serif;
  font-size: 4.5rem;
  font-weight: 600;
  letter-spacing: 0.04em;
  margin-top: 4.6rem;
  @media (max-width: 575.98px) {
    font-size: 3.52rem;
  }

  // Small devices (landscape phones, 576px and up)
  @media (min-width: 576px) and (max-width: 767.98px) {
    font-size: 3.52rem;
  }

  @media (min-width: 768px) and (max-width: 991.98px) {
    font-size: 3.52rem;
  }
  @media (max-width: 1024px) {
    font-size: 3.52rem;
  }
}

.contact-box {
  position: absolute;
  width: 80rem;
  height: 90rem;
  left: 47rem;
  align-content: center;
  top: 150rem;
  background: #000;

  @media (min-width: 320px) and (max-width: 667px) {
    top: 190rem;
    left: 10rem;
    height: 90rem;
  }
  @media (min-width: 667px) and (max-width: 812px) {
    top: 200rem;
    left: 10rem;
    height: 90rem;
  }
  @media (min-width: 812px) and (max-width: 1024px) {
    top: 140rem;
    left: 27rem;
    height: 90rem;
  }
  @media (min-width: 1024px) and (max-width: 1366px) {
    top: 140rem;
    left: 27rem;
    height: 90rem;
  }
}

.contact-box .contact-container-border-line {
  background-color: #fff;
  position: absolute;
  top: 1rem;
  left: 37.096rem;
  width: 4.72rem;
  height: 0.288rem;

  @media (min-width: 320px) and (max-width: 667px) {
    left: 36rem;
  }
}

.contact-container .primary-location {
  text-align: center;
  font-family: "Roboto", sans-serif;
  font-size: 2.5rem;
  font-weight: 600;
  letter-spacing: 0.04em;
  color: #fff;
  margin-top: 2.8rem;
}
.contact-container .primary-info {
  text-align: center;
  font-family: "Roboto", sans-serif;
  font-size: 2.5rem;
  font-weight: 600;
  letter-spacing: 0.04em;
  color: #fff;
}
.contact-container .primary-thank-you-note {
  text-align: center;
  font-family: "Roboto", sans-serif;
  font-size: 2.5rem;
  font-weight: 600;
  letter-spacing: 0.04em;
  color: #fff;
  margin-top: 3rem;
}
a.primary-direction-route {
  font-family: "Josefin Sans", sans-serif !important;
  font-size: 2.5rem;
  font-weight: 600;
  letter-spacing: 0.04em;
  color: #e294c5 !important;
  margin-left: 27rem !important;
  text-transform: uppercase;
  text-align: center;
}
a.primary-direction-route:hover {
  color: #fff !important;
}

.contact-container .secondary-header {
  text-align: center;
  text-transform: uppercase;
  font-family: "Josefin Sans", sans-serif;
  font-size: 3rem;
  font-weight: 600;
  letter-spacing: 0.04em;
  margin-top: 4.6rem;
  color: #f8e7f2;
}

.contact-info {
  margin-top: 2.8rem;
  top: 31rem;
  width: 50rem;
  height: 4rem;
  font-size: 1.7rem;
  color: #000;
  margin-left: 6.3rem;
  font-family: "Roboto", sans-serif;
}
.contact-row {
  width: 80%;
  margin-left: 8rem;
  align-content: center;
}

.btn {
  background-color: #e294c5 !important;
  border: 1px solid #e294c5 !important;
  border-radius: 1rem;
}

.btn:hover {
  background: #f8e7f2;
  border: 1px solid #f8e7f2;
}
