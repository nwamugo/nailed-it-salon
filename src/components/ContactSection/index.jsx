import React from 'react';

import './ContactSection.scss';

const ContactSection = () => {
  return (
    <div className="contact-box" id="contactus">
      <hr className="contact-container-border-line" />
      <div className="contact-container">
        <h2 className="primary-header">Contact Us</h2>
        <h3 className="primary-location">
          Ground Foor, Al Raed Groove, Ashirat Sudayr street, &nbsp;
          <br />
          Ar Raid, Riyadh 12355
        </h3>
        <h3 className="primary-info">
          info@mysite.com&nbsp;&nbsp;|&nbsp;&nbsp;+966 56 057 7709
        </h3>
        <h2 className="secondary-header">Opening Hours</h2>
        <h3 className="primary-info">
          Mon - Thur &amp; Sat: 11am - 9pm
          <br />
          Friday: 2pm - 9pm
          <br />
          Sunday: Closed
        </h3>
        <div>
          <form>
            <div className="contact-row form-row align-items-center">
              <div className="col">
                <input
                  type="text"
                  className="form-control contact-info"
                  id="inlineFormInputName"
                  placeholder="Name"
                />
              </div>
              <div className="col">
                <input
                  type="text"
                  className="form-control contact-info"
                  id="inlineFormInputEmail"
                  placeholder="Email"
                />
              </div>
              <div className="col">
                <input
                  type="text"
                  className="form-control contact-info"
                  id="inlineFormInputPhoneNumber"
                  placeholder="Phone Number"
                />
              </div>
              <div className="col">
                <textarea
                  className="form-control contact-info"
                  id="inlineFormInputeMessage"
                  rows="3"
                  placeholder="Message"
                />
              </div>
              <div className="col-auto">
                <button type="submit" className="btn btn-primary contact-info">
                  Submit
                </button>
              </div>
            </div>
          </form>
        </div>
        <h3 className="primary-thank-you-note">Thank You For Contacting Us</h3>
        <a
          href="/directions"
          className="primary-direction-route"
          rel="noopener noreferrer"
        >
          See Directions &gt;
        </a>
      </div>
    </div>
  );
};

export default ContactSection;
