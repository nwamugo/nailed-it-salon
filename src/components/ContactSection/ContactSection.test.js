import React from 'react';
import { shallow } from 'enzyme';
import ContactSection from './index';

describe('ContactSection', () => {
  it('should not have regressed', () => {
    const component = shallow(<ContactSection />);

    expect(component).toMatchSnapshot();
  });
});
