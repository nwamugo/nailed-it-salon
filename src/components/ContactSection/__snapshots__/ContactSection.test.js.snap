// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactSection should not have regressed 1`] = `
<div
  className="contact-box"
>
  <hr
    className="contact-container-border-line"
  />
  <div
    className="contact-container"
  >
    <h2
      className="primary-header"
    >
      Contact Us
    </h2>
    <h3
      className="primary-location"
    >
      Ground Foor, Al Raed Groove, Ashirat Sudayr street,  
      <br />
      Ar Raid, Riyadh 12355
    </h3>
    <h3
      className="primary-info"
    >
      info@mysite.com  |  Tel: 123-456-7890
    </h3>
    <h2
      className="secondary-header"
    >
      Opening Hours
    </h2>
    <h3
      className="primary-info"
    >
      Mon - Thur & Sat: 11am - 9pm
      <br />
      Friday: 2pm - 9pm
      <br />
      Sunday: Closed
    </h3>
    <div>
      <form>
        <div
          className="contact-row form-row align-items-center"
        >
          <div
            className="col"
          >
            <input
              className="form-control contact-info"
              id="inlineFormInputName"
              placeholder="Name"
              type="text"
            />
          </div>
          <div
            className="col"
          >
            <input
              className="form-control contact-info"
              id="inlineFormInputEmail"
              placeholder="Email"
              type="text"
            />
          </div>
          <div
            className="col"
          >
            <input
              className="form-control contact-info"
              id="inlineFormInputPhoneNumber"
              placeholder="Phone Number"
              type="text"
            />
          </div>
          <div
            className="col"
          >
            <textarea
              className="form-control contact-info"
              id="inlineFormInputeMessage"
              placeholder="Message"
              rows="3"
            />
          </div>
          <div
            className="col-auto"
          >
            <button
              className="btn btn-primary contact-info"
              type="submit"
            >
              Submit
            </button>
          </div>
        </div>
      </form>
    </div>
    <h3
      className="primary-thank-you-note"
    >
      Thank You For Contacting Us
    </h3>
    <a
      className="primary-direction-route"
      href="#"
      rel="noopener noreferrer"
      target="_blank"
    >
      See Directions &gt;
    </a>
  </div>
</div>
`;
